pkgs/development/libraries/cpp-utilities/default.nix

```diff
@@ -18,19 +19,23 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
   nativeCheckInputs = [ cppunit ];
+  buildInputs = [
+    iconv # needed on Darwin
```

Comment:
Then maybe we should add this conditionally? Is upstream aware of this dependency?

Reply:
Seems like upstream [is aware of thr dependency](https://github.com/Martchus/cpp-utilities/issues/4), at least on Darwin

Reply:
> Seems like upstream [is aware of the dependency](https://github.com/Martchus/cpp-utilities/issues/4), at least on Darwin

Thanks for the link, I'd put that link with an explanation as a comment to this buildInput, and add it only conditionally (`lib.optionals (stdenv.isDarwin)`).

Reply:
Should be done, made `iconv` optional in both `cpp-utilities` and `syncthingtray`.